app/controllers/passengers_controller.rb

```diff
@@ -40,6 +41,7 @@ def create
 
   def update
     if @passenger.update(passenger_params)
+      @passenger.doctors_note.update_attributes(overridden_by: @current_user)
```

Comment:
We don't want to do this every time someone saves a passenger, right? Only when someone's specifically in the process of overriding the doctor's note.